### PR TITLE
Upgrade to new ways

### PR DESCRIPTION
**Pull request by regisphilibert**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Forestry needs a config.yaml at the root.
+# Is overwriten by config/_default/config.yaml
+# ###################################
+forestry: Here you go
```
Comment (DirtyF) [inline config.yaml:4]: 😄

**Comment by DirtyF** [comment]
This page needs some updates, I'll submit a PR.
<details><summary>Browser metadata</summary>





```
Path:      /tool/tinacms/
Browser:   Chrome 91.0.4472.77 on Mac OS 10.15.7
Viewport:  1440 x 825 @2x
Language:  fr-FR
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-121--thenewdynamic-org.netlify.app%2Ftool%2Ftinacms%2F&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1440x825&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-121--thenewdynamic-org.netlify.app/tool/tinacms/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b7f5209459710007fa7a82&commentID=60b7f702b3c82f48013fa618&resolution=resolved)

**Comment by regisphilibert** [comment]
Can you base it from this branch @DirtyF ?

**Comment by regisphilibert** [comment]
@budparr This is ready for your evaluation. There are a few styling changes you should look at (nothing major mostly spacing and line-height et...) but one I can't explain yet is the change in viewport breakpoints... For some reason the new site breaks later for `md`...

**Comment by regisphilibert** [comment]
@DirtyF I updated the Tina entry on master so this does not have to wait for this PR resolution. (my bad for asking to use this branch as your PR's base.

**Comment by DirtyF** [comment]
Really awesome to see how you leverage Hugo modules in there 👏

**Comment by regisphilibert** [comment]
Thanks @DirtyF we'd like to turn this into something easier to use for everyone, a framework in the longterm, a clean non opinionated starter in the short term. For now it's a GitHub template repo: https://github.com/theNewDynamic/hugo-project-template 

Feel free to give it a try!

**Comment by DirtyF** [comment]
Showcase thumbnails are not displayed in the preview 🤔 
<details><summary>Browser metadata</summary>





```
Path:      /showcase/
Browser:   Firefox 89.0 on Mac OS 10.15
Viewport:  1440 x 815 @2x
Language:  fr
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-121--thenewdynamic-org.netlify.app%2Fshowcase%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=89.0&resolution=1440x815&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-121--thenewdynamic-org.netlify.app/showcase/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b8f455e024fb000889eb0a&commentID=60b8f5b11dc96f1736373052&resolution=resolved)

**Comment by regisphilibert** [comment]
Good catch @DirtyF

**Comment by regisphilibert** [comment]
And here I was looking for a good use case on that project for [tnd-media](https://github.com/theNewDynamic/hugo-module-tnd-media) 

https://github.com/theNewDynamic/thenewdynamic.org/blob/6d273b2d524cea6403c728a1e94d2c5f598bde41/layouts/showcase/summary.html#L4-L5

**Comment by budparr** [comment]
This looks fine to me. I'm not terribly concerned about the state of it, because I'm going to do some refreshing when I add the podcast section.

**Review by budparr — APPROVED** [review]

**Comment by regisphilibert** [comment]
Ok, I'll make this the production branch then.

**Pull request closed without merging**